src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -1314,6 +1314,7 @@ def warn_qlora_zero3_w_use_reentrant(cls, data):
             and data.get("gradient_checkpointing_kwargs", {})
             and data.get("gradient_checkpointing_kwargs", {}).get("use_reentrant")
             is False
+            and data.get("deepspeed", "") is not None
             and "zero3" in data.get("deepspeed", "")
```

Comment:
isn't this redundant with the next line? if it's empty, doesn't the next line `data.get("deepspeed", "")` default it to an empty string?

Reply:
when deepspeed is not specified, `data.get("deepspeed", "")` returns `None` and I get `TypeError: argument of type 'NoneType' is not iterable`